client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -610,6 +610,15 @@ export class CheckoutThankYou extends Component<
 				);
 			}
 
+			const gSuiteOrExtraLicenseOrGoogleWorkspace = purchases.find(
+				isGSuiteOrExtraLicenseOrGoogleWorkspace
+			);
+			if ( gSuiteOrExtraLicenseOrGoogleWorkspace ) {
```

Comment:
@nightnei Is there any reason why we wouldn't add a `elseif` block to the above conditions?

```
} else if ( purchases.find( isGSuiteOrExtraLicenseOrGoogleWorkspace ) ) { 
```